sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeDisableAutomaticCheckForUpdateTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.distribute;
+
+import com.microsoft.appcenter.http.HttpClient;
+import com.microsoft.appcenter.http.ServiceCallback;
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static com.microsoft.appcenter.distribute.DistributeConstants.PREFERENCE_KEY_REQUEST_ID;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+public class DistributeDisableAutomaticCheckForUpdateTest extends AbstractDistributeTest {
+
+    @Test
+    public void disableAutomaticCheckForUpdateBeforeDistributeStartDoesNotCheckForUpdate() {
```

Comment:
This is a test for Public track. We should also make sure it doesn't open a browser on private track.

Reply:
This test has nothing to do with if a browser open or not